cmake/reports/CMakeLists.txt

```
@@ -17,3 +17,18 @@ foreach(report ram_report rom_report)
             $<TARGET_PROPERTY:zephyr_property_target,${report}_DEPENDENCIES>
     )
 endforeach()
+
+find_program(PUNCOVER puncover)
+
```

Could you safeguard this around
```suggestion
if(NOT ${PUNCOVER} STREQUAL PUNCOVER-NOTFOUND)
```